nailgun/entity_fields.py

```diff
@@ -198,10 +200,32 @@ def gen_value(self):
 class DateField(Field):
     """Field that represents a date"""
 
+    def __init__(self, min_date=None, max_date=None):
+        # If ``None`` is passed then ``FauxFactory`` will deal with it.
+        self.min_date = min_date
+        self.max_date = max_date
+
+    def gen_value(self):
+        """Return a value suitable for a :class:`DateField`."""
+        # pylint:disable=no-self-use
+        # See comment near top of module.
```

Comment:
You should be able to drop these two lines of commenting. This method does make use of `self`.